src/components/graph/search/GraphSearchDialog.tsx

```diff
@@ -195,6 +256,11 @@ const GraphSearchDialog = (props: Props) => {
 
 	const fetchObjectById = (id: string, abortController: AbortController) => {
 		const searchTimestamp = moment.utc().valueOf();
+		if (
```

Comment:
if you replace `onValueChange` to `onSearchSuccess` (see above), you can remove this condition

Reply:
Even with change this condition prevents unneeded requests even with change to onValueChange and it prevents requests when in history mode inputted date

Reply:
ok. can we move this condition to `onValueChange` before calling `fetchObjectById`? if yes, let's do it. and create new function for this logic like `isTimestamp(value: string): boolean` in `src/helpers/date.ts` and use it.